core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/loadbalancer/LoadBalancerResponse.java

```diff
@@ -0,0 +1,85 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.loadbalancer;
+
+import java.io.Serializable;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import io.swagger.annotations.ApiModelProperty;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.sequenceiq.common.api.type.LoadBalancerType;
+
+@JsonInclude(Include.NON_NULL)
+public class LoadBalancerResponse implements Serializable {
+
+    @ApiModelProperty
+    private String fqdn;
+
+    @ApiModelProperty
+    private String cloudDns;
```

Comment:
Can we have model description for the new request/response objects?
It would be easier to understand what value the field could contain.

An example on StackV4Response: https://github.com/hortonworks/cloudbreak/blob/5d33992d0bc8b2a278a6c5ecf45a0876534a262c/core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/StackV4Response.java#L82

Reply:
Done.